restore/harbor/restore-harbor-job-azure.yaml

```diff
@@ -42,8 +42,6 @@ spec:
                   key: azure-account-name
             - name: STORAGE_TYPE
               value: "azure"
-            - name: S3_BUCKET
-              value: ${S3_BUCKET}
```

Comment:
@lunkan93 @Elias-elastisys  This looked out of place but maybe it was added intentionally? Does the script not work without it?

Reply:
This was accidentally left when the s3 job file was copied.